src/NServiceBus.Core/Unicast/MessageHandlerRegistry.cs

```diff
@@ -26,10 +26,6 @@ internal MessageHandlerRegistry(Conventions conventions)
         public List<MessageHandler> GetHandlersFor(Type messageType)
         {
             Guard.AgainstNull(nameof(messageType), messageType);
-            if (!conventions.IsMessageType(messageType))
-            {
-                return noMessageHandlers;
```

Comment:
Looks like this is the only place that `nonoMessageHandlers` was used, so the static field can be removed.

Reply:
Fixed